docs/guides/importing.md

```diff
@@ -22,6 +22,7 @@ resource "packetfabric_cloud_router" "awesome_cloud_routers" {
   provider     = packetfabric
   asn          = 4556
   name         = "Awesome Cloud Routers"
+  account_uuid = var.pf_account_uuid
```

Comment:
@pontovinte @rjouhann Not sure why doc-gen adds this.. If I recall this wasn't required. Please confirm

Reply:
`account_uuid` is a required parameter, however, I added the option to set it up with an env variable in v0.4.2 so we could actually omit it from the example (https://github.com/PacketFabric/terraform-provider-packetfabric/pull/272)

Reply:
@elXaptation I would just remove `account_uuid = var.pf_account_uuid` from the examples.